packages/docs-site/src/components/presenters/sketch-widget/sketch-widget.scss

```diff
@@ -0,0 +1,77 @@
+@import "@royalnavy/css-framework";
+
+$border-radius: 4px;
+$head-line-height: calc(50px - 2rem);
+$body-line-height: calc(64px - 2rem);
+
+.sketch-widget {
+  border: 1px solid #dae2ec;
+  color: #253b5b;
+  border-radius: $border-radius;
+  box-shadow: 0 1px 2px 0 rgba(000, 000, 000, .15);
+}
+
+.sketch-widget__head {
+  display: flex;
+  flex-flow: column;
+  background-color: #f0f4f8;
+  padding: spacing(4);
+  border-top-left-radius: $border-radius;
+  border-top-right-radius: $border-radius;
+}
+
+.sketch-widget__title {
+  line-height: $head-line-height;
+  margin-bottom: spacing(4);
+}
+
+.sketch-widget__link {
+  line-height: $head-line-height;
+  color: #3a8fdd;
+  text-decoration: none;
+
+  svg {
```

Comment:
do we want to style by tag name? would it be better to use a class name?

Reply:
Hmm well I coupled it very closely to the parent element. This is up for debate I think.